taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from django.urls import path, include
+from taxi.views import index
```

Comment:
Separate different types of imports in this order: standard library, external library, custom modules.